clients/cypress-e2e/cypress/e2e/smoke_test.cy.ts

```diff
@@ -57,23 +57,6 @@ describe("Smoke test", () => {
     });
   });
 
-  it("can access Mongo and Postgres connectors from the Admin UI", () => {
-    cy.intercept(`${API_URL}/connection_type`).as("getConnectionType");
-    cy.intercept(`${API_URL}/connection*`).as("getConnections");
-
-    cy.visit(ADMIN_UI_URL);
-    cy.login();
-    cy.get("div").contains("Configure privacy requests").click();
```

Comment:
I think we should still keep this test around, but get to this page through the nav bar instead of through the home page tile. something like `cy.contains("nav a", "Privacy requests")` instead

Reply:
We are doing away with the connectors subtab under Privacy. We will only have linked connectors under systems.

Reply:
I added the test back but there will need to be another PR when the page is removed to seed system data and accessing the connector through the system.

Reply:
ah got it, thanks for the context and for adding the test back! 🙏